lib/src/repositories/merchants_repository.dart

```diff
@@ -0,0 +1,32 @@
+import 'package:cartan/src/models/merchant.dart';
+import 'dart:convert';
+import 'package:flutter/services.dart' show rootBundle;
+
+class MerchantsRepository {
+  List<Merchant> _merchants = [];
+  bool _initialized = false;
+
+  Future<void> initialize() async {
+    if (_initialized) return;
```

Comment:
mete sempre {}, mesmo que seja só o1 liner,

```suggestion
    if (_initialized) {
        return;
    } 
```

Reply:
feito